### DISCUSSION TITLE
How to remove rust desk client from macOS

### DISCUSSION THREAD

**Discussion by ibrahimakgn** [rustdesk/rustdesk#3475] [General]
Hello,

I could not find any info for removing rustDesk client from macOs. Is there any idea?

**Comment by jarrah31** [comment]
I too would like to remove RustDesk from MacOS due to the Lock Screen bug which makes this unusable.

**Comment by jrop** [comment]
It appears that there is an uninstaller script for macOS here:

https://github.com/rustdesk/rustdesk/blob/master/src/platform/privileges_scripts/uninstall.scpt

...which is equivalent to the following shell commands that you can run in a terminal window:

```sh
sudo launchctl unload -w /Library/LaunchDaemons/com.carriez.RustDesk_service.plist
sudo rm /Library/LaunchDaemons/com.carriez.RustDesk_service.plist
sudo rm /Library/LaunchAgents/com.carriez.RustDesk_server.plist
launchctl remove com.carriez.RustDesk_server
```

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/discussions/5631#discussioncomment-6943538

> run with sudo 
> ```
> launchctl unload -w /Library/LaunchDaemons/com.carriez.RustDesk_service.plist
> /bin/rm /Library/LaunchDaemons/com.carriez.RustDesk_service.plist
> /bin/rm /Library/LaunchAgents/com.carriez.RustDesk_server.plist
> ```
> 
> then run
> 
> ```
> launchctl remove com.carriez.RustDesk_server
> ```

**Reply by C-Duv** [reply]
@rustdesk but the `sudo launchctl unload -w /Library/LaunchDaemons/com.carriez.RustDesk_service.plist` command fails with:

```
Unload failed: 5: Input/output error
Try running `launchctl bootout` as root for richer errors.
```

**Comment by vvbrain** [comment]
When I try to uninstall rust desk on Mac,it tells me that it was running now, i can not remove it from my Mac.But, where can I stop it from running? @thornzz This helped me.

**Comment by rustdesk** [comment]
![image](https://github.com/rustdesk/rustdesk/assets/71636191/5b97080a-c13d-4d44-b0c7-c41068498f05)

or

![image](https://github.com/rustdesk/rustdesk/assets/71636191/5b3938f8-b93c-47d6-ad90-877726918e37)

**Comment by codiblau** [comment]
I have deleted this app. Just delete this:

Completed removal:
•  Main application: /Applications/RustDesk.app
•  Preferences folder: ~/Library/Preferences/com.carriez.RustDesk (including all configuration files)
•  Log files: ~/Library/Logs/RustDesk (including all log files)

**Reply by codiblau** [reply]
You can check before with this command: sudo find ~/Library -name "*rustdesk*" -o -name "*RustDesk*" 2>/dev/null
